library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java

```diff
@@ -150,7 +150,7 @@ public void onCreate() {
 
         downloadClientReadyChecker = getDownloadClientReadyChecker();
 
-        mNotifier = new DownloadNotifier(this, getNotificationImageRetriever(), getResources());
+        mNotifier = DownloadNotifier.newInstance(this, getNotificationImageRetriever(), getResources());
```

Comment:
I don't think we're using this pattern in this project.

Can you just new everything up here to make it explicit what is happening